### PR TITLE
Hiding sidebar if empty - PM4EH-98

### PR DESCRIPTION
**Pull request by paulosakamoto**
## Issue & Reproduction Steps
It is necessary to hide the sidebar in some routes if the user is not an administrator.

It was working until version PM 4.1.27

This issue was reported in [PM4EH-98](https://processmaker.atlassian.net/browse/PM4EH-98).

## Solution
The only modification made in this project is if the `$sidebar` has the `display === false` option the sidebar will not be displayed.

It will be up to the other project ( [adps](https://github.com/ProcessMaker/package-zj-adps) ) to decide in which situations the menu items should be hidden.

## How to Test
- Install the package [adps](https://github.com/ProcessMaker/package-zj-adps), branch bugfix/PM4EH-98
- Create a non-admin user and give full access to requests, tasks and users
- Navigate between the Request, Task and Admin tabs.
- A non-admin should not have a sidebar on the Requests and Tasks tabs, but should see the sidebar on the Admin tab

## Related Tickets & Packages
- [PM4EH-98](https://processmaker.atlassian.net/browse/PM4EH-98)

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

### CODE REVIEW

**Review by danloa — COMMENTED** [review]
@paulosakamoto  I've being testing the solution but I see that probably there is a more complex problem: Laravel processes package Middlewares in alphabetical order. 

I tested with data-sources, saved-search, and translations packages. 
If the ADPS middleware code is put in data-sources, saved-search overwrites those changes and the side bar menus are displayed.

When I put the ADPS middleware in translations, it works ok because translations overwrites saved-search changes and the menus are removed.

As the package is called ADPS, it is processed first by Laravel and the subsequent packages overwrite its changes...

In short: we need to investigate so that middlewares are processed in an expected order (May be to implement a "order position" variable in Middlawares?) Probably Laravel has a way to implement this.

**Comment by paulosakamoto** [comment]
The implementations will be done only in the adps package.
